README.md

```diff
@@ -147,6 +147,13 @@ The configuration file defines:
 
 - Project name: the `<project-slug>` name of the Project
 - The DICOM dataset modalities to retain (e.g. `["DX", "CR"]` for X-Ray studies)
+- The minimum number of instances required by a series (defaults to 1). May be set higher than 1 to filter out
```

Comment:
should `min_instances_per_series` default to 2?

Reply:
Yeah I think so